sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -4,22 +4,205 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
 
 namespace Azure.Iot.Hub.Service
 {
     /// <summary>
-    /// Jobs Client place holder
+    /// Jobs client to support Import/Export and Scheduled Jobs.
```

Comment:
Could you expand on this comment a bit? As a user, I would expect this header to explain what import/export jobs are, what scheduled jobs are, and how they are different (different quotas on hub side, for instance)